app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -40,16 +40,19 @@ class MainActivity : AppCompatActivity() {
         var movie8: Movie = Movie("The Boondock Saints", "Fantasy", "Jack Black", "https://m.media-amazon.com/images/M/MV5BYzVmMTdjOTYtOTJkYS00ZTg2LWExNTgtNzA1N2Y0MDgwYWFhXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_.jpg", "A movie about a blue person.");
         var movie9: Movie = Movie("Training Day", "Fantasy", "Jack Black", "https://images-na.ssl-images-amazon.com/images/I/91EbJZH9JWL._SL1500_.jpg", "A movie about a blue person.");
         var movie10: Movie = Movie("Smurfs", "Fantasy", "Jack Black", "https://upload.wikimedia.org/wikipedia/en/1/11/TheSmurfs2011Poster.jpg", "A movie about a blue person.");
-        testMovieData.add(movie1);
-        testMovieData.add(movie2);
-        testMovieData.add(movie3);
-        testMovieData.add(movie4);
-        testMovieData.add(movie5);
-        testMovieData.add(movie6);
-        testMovieData.add(movie7);
-        testMovieData.add(movie8);
-        testMovieData.add(movie9);
-        testMovieData.add(movie10);
+        with(testMovieData){
```

Comment:
Implemented with() since the same method was being invoked on testMovieData repeatedly. Abiding by Kotlin Best Practices. 


https://phauer.com/2017/idiomatic-kotlin-best-practices/

Reply:
A good suggestion, though this was also suggested in a previous code review.  In that case, the first one in wins.  

I'd go one step further.  Should data be created and stored in an Activity?  Probably not.  Move this test data to another class.